docs/resources/outbound_callabletimeset.md

```diff
@@ -0,0 +1,83 @@
+---
+page_title: "genesyscloud_outbound_callabletimeset Resource - terraform-provider-genesyscloud"
+subcategory: ""
+description: |-
+  Genesys Cloud outbound callabletimeset
+---
+# genesyscloud_outbound_callabletimeset (Resource)
+
+Genesys Cloud outbound callabletimeset
+
+## API Usage
+The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Client has been granted the necessary scopes and permissions to perform these operations:
+
+* [GET /api/v2/outbound/callabletimesets](https://developer.genesys.cloud/routing/outbound/#get-api-v2-outbound-callabletimesets)
+* [POST /api/v2/outbound/callabletimesets](https://developer.genesys.cloud/routing/outbound/#post-api-v2-outbound-callabletimesets)
+* [DELETE /api/v2/outbound/callabletimesets/{callableTimeSetId}](https://developer.genesys.cloud/routing/outbound/#delete-api-v2-outbound-callabletimesets--callableTimeSetId-)
+* [GET /api/v2/outbound/callabletimesets/{callableTimeSetId}](https://developer.genesys.cloud/routing/outbound/#get-api-v2-outbound-callabletimesets--callableTimeSetId-)
+* [PUT /api/v2/outbound/callabletimesets/{callableTimeSetId}](https://developer.genesys.cloud/routing/outbound/#put-api-v2-outbound-callabletimesets--callableTimeSetId-)
+
+
+## Example Usage
+
+```terraform
+resource "genesyscloud_outbound_callabletimeset" "example-callable-time-set" {
+  name = "Example Callable time set"
+  callable_times {
+    time_zone_id = "Africa/Abidjan"
+    time_slots {
+      start_time = "07:00:00"
```

Comment:
Did you say you needed to add the milliseconds to fix that `plan not empty` error? If so, it'd be worth reflecting that in the example.